build.sbt

```diff
@@ -224,6 +227,28 @@ lazy val http4s = project
   )
   .dependsOn(core)
 
+lazy val zioHttp = project
+  .in(file("adapters/zio-http"))
+  .settings(scalaVersion := scala13)
```

Comment:
Hm, that doesn't quite work either. It seems to get hung up on not finding `io.d11:zhttp_2.12:1.0.0.0-RC15+33-c11af52b-SNAPSHOT`, which makes sense since it's only published for 2.13.

And

```
  .settings(commonSettings)
  .settings(
    scalaVersion := scala13,
    crossScalaVersions := Seq(scala13)
  )
```

fails with the same error about not finding `com.github.ghostdogpr:caliban_2.13:0.9.5+71-07eee183+20210507-1531-SNAPSHOT`. 🤷

Reply:
Even if you remove `scalaVersion := scala13 `?

Reply:
Hmm, removing it might not make sense actually since the default is 2.12. Weird 🤔

Reply:
Yes, if I remove `scalaVersion := scala13` it fails on not being able to find zio-http for Scala 2.12.

Reply:
But I find it a bit weird that it tries to resolve it from Sonartype. Maybe it's because I haven't cross compiled `core` locally? I'll try to give that a go!

Reply:
I think it's ok actually. Locally you can do `++2.13.5` in sbt to switch to 2.13. I think the CI will pass.